app/frontend/src/views/file/Download.vue

```diff
@@ -45,11 +45,7 @@ export default {
     async getFile(fileId) {
       this.showDownloadLink = false;
       await this.downloadFile(fileId);
-      if (
-        this.downloadedFile &&
-        this.downloadedFile.data &&
-        this.downloadedFile.headers
-      ) {
+      if (this.downloadedFile && this.downloadedFile.headers) {
```

Comment:
This allows the download to begin if there is no data inside of the file to be downloaded, no submissions permitted. This way the user is actually notified that something happened instead of it just spinning infinitely.